tests/test_human_data_cache.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+
+from mmhuman3d.data.data_structures.human_data import HumanData
+from mmhuman3d.data.data_structures.human_data_cache import (
+    HumanDataCacheReader,
+    HumanDataCacheWriter,
+)
+
+human_data_cache_path = 'tests/data/human_data/human_data_cache.npz'
+
+
+def test_write():
+    human_data = HumanData.new(key_strict=False)
+    human_data['keypoints2d'] = np.ones(shape=(100, 199, 3))
+    human_data['keypoints2d_mask'] = np.ones(shape=(199, ))
```

Comment:
May consider using [`get_keypoint_num()`](https://github.com/open-mmlab/mmhuman3d/blob/e58b0738616fa80124b8f3f1e66af6300c6f3f79/mmhuman3d/core/conventions/keypoints_mapping/__init__.py#L307) as the number of keypoints may change in the future.